contrib/scripts/install_prereq

```diff
@@ -190,9 +189,11 @@ in_test_mode() {
 
 check_installed_debs() {
 	for pack in "$@" ; do
-		tocheck="${tocheck} ^${pack}$ ~P^${pack}$"
+		EXISTS=`dpkg -l | cut -d ' ' -f 3 | grep ^"${pack}"\\$ | wc -l`
+		if [ ${EXISTS} -eq 0 ]; then
+		  echo "${pack}"
+    fi
```

Comment:
```
if ! dpkg -l | awk 'NR > 5 {print $2}' | grep -q ^"${pack}"$; then
    echo "${pack}"
fi
```

Reply:
Actually, it may be as simple as:

`dpkg -l "${pack}" || echo "${pack}"`

Reply:
Hey @seanbright -- your suggestion here isn't quite safe:

```
cat test.sh

#!/usr/bin/env sh

PACK="SOMETHING_I_DONT_HAVE"

EXISTS=`dpkg -l "${pack}" >/dev/null 2>&1 || echo "${pack}"`
if [ ${EXISTS} -eq 0 ]; then
  echo "${pack}"
fi

./test.sh
./test.sh: 6: [: -eq: unexpected operator


```

The approach I have is guaranteed to provide a numeric output with the `| wc -l`

Reply:
I think you misunderstood my suggestion. I was suggesting this:

```shell
check_installed_debs() {
  for pack in "$@" ; do
    dpkg -l "${pack}" >/dev/null 2>&1 || echo "${pack}"
  done
}
```